x-pack/plugins/ml/readme.md

```diff
@@ -0,0 +1,145 @@
+# Documentation for ML UI developers
+
+## Requirements
+
+To use machine learning features, you must have Platinum or Enterprise license or a free 14-day trial. File Data Visualizer requires Basic license.
```

Comment:
Might be worth adding a link to https://www.elastic.co/guide/en/machine-learning/current/setup.html too?